contributing/code_of_conduct/index.rst

```diff
@@ -0,0 +1,91 @@
+Code of Conduct
+===============
+
+Our Pledge
+----------
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
```

Comment:
It's maybe a cultural difference, but are we sure we want to use the "race" word?

Quoting Wikipedia:

> Since the second half of the 20th century, the association of race with the ideologies and theories of scientific racism has led to the use of the word race itself becoming problematic. Although still used in general contexts, race has often been replaced by less ambiguous and loaded terms: populations, people(s), ethnic groups, or communities, depending on context.

Reply:
See also: https://github.com/ContributorCovenant/contributor_covenant/issues/476

Reply:
We could follow the same practices of European bureaucrats and replace `ethnicity` and `race` by just `ethnic origin`. It conveys the same meaning but it's less controversial for some people.